skimage/measure/_marching_cubes_lewiner.py

```diff
@@ -154,6 +154,7 @@ def marching_cubes_lewiner(volume, level=None, spacing=(1., 1., 1.),
     # Apply algorithm
     # Check if a mask array is passed
     if mask is not None:
+        assert mask.shape == volume.shape, 'volume and mask must have the same shape.'
```

Comment:
We typically prefer raising a `ValueError` instead of using asserts.

Reply:
Refactored